src/ir_Daikin.cpp

```diff
@@ -2323,12 +2325,12 @@ uint8_t IRDaikin176::getFan() {
 
 // Convert a standard A/C Fan speed into its native fan speed.
 uint8_t IRDaikin176::convertFan(const stdAc::fanspeed_t speed) {
-     switch (speed) {
+  switch (speed) {
     case stdAc::fanspeed_t::kMin: return kDaikinFanMin;
-    case stdAc::fanspeed_t::kLow: return kDaikinFanMin + 1;
-    case stdAc::fanspeed_t::kMedium: return kDaikinFanMin + 2;
-    case stdAc::fanspeed_t::kHigh: return kDaikinFanMax - 1;
-    case stdAc::fanspeed_t::kMax: return kDaikinFanMax;
+    case stdAc::fanspeed_t::kLow:
```

Comment:
This bit looks wrong to me.
You are returning `kDaikinFanMax - 2` for all of Low, Medium, High, & Max. 

You possibly want:
```
switch (speed) {
  case stdAc::fanspeed_t::kMin:
  case stdAc::fanspeed_t::kLow:
    return kDaikinFanMin;
  case stdAc::fanspeed_t::kMedium:
    return kDaikinFanMin + 1;
  case stdAc::fanspeed_t::kHigh:
  case stdAc::fanspeed_t::kMax:
    return kDaikin176FanMax;
  ...
```